inst/rmarkdown/templates/springer/resources/template.tex

```diff
@@ -132,7 +132,8 @@
 
 $body$
 
-\bibliographystyle{$if(biblio-style)$$biblio-style$$else$spbasic$endif$}
+
+\bibliographystyle{$if(biblio-style)$$biblio-style$$elseif(bibstyle)$$bibstyle$$else$spbasic$endif$}
```

Comment:
Yes it would but unfortunately for us `elsif` is only supported since Pandoc 2.8
https://pandoc.org/releases.html#pandoc-2.8-2019-11-22

So we need to use good old nested if else structure.

Reply:
![image](https://user-images.githubusercontent.com/8617595/102384229-3a991300-3fab-11eb-90b6-8d29f91fb7d1.png)